ckanext/dms/helpers.py

```diff
@@ -84,3 +84,15 @@ def get_all_groups():
     return logic.get_action('group_list')(
             data_dict={'sort': 'title asc', 'all_fields': True})
 
+
+def get_site_statistics() -> dict[str, int]:
+    '''This function used be a core helper but it was removed in CKAN 2.11.0
+    We reproduce it here to templates can continue working as usual.
+    '''
+    stats = {}
+    stats['dataset_count'] = logic.get_action('package_search')(
```

Comment:
Use the toolkit instead... 

```suggestion
    stats['dataset_count'] = toolit.get_action('package_search')(
```